<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminaddbook</class>
 <widget class="QDialog" name="adminaddbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="gridFrame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
background: rgb(255, 255, 255);
fontsize: 15px;
font-family: Century Gothic, sans-serif;
}

QFrame{
border: solid 10px rgba(0, 0, 0);
}

QLineEdit{
color: #8d98a1;
background-color: #d3d3d3;
font-size: 18px;
border-style: outset;
border-radius: 10px;
}
QPushButton{
background: #ced1d8;
border-style: outset;
border-radius: 5px;
}
QPushButton:pressed{
background-color: rgb(77, 77, 77);
border-style: inset;
}
</string>
   </property>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

border-width: 1px;

border-color: black;

border-radius: 10px;

border-style: outset;

font: 14px;

padding:2px;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px</string>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 34px</string>
    </property>
    <property name="text">
     <string>Add Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="canceladd">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px</string>
    </property>
    <property name="text">
     <string>Title:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmadd">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Changes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px</string>
    </property>
    <property name="text">
     <string>Image:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="book_name_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="book_author_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>yoobee image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px</string>
    </property>
    <property name="text">
     <string>Copies:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="book_copies_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QCheckBox" name="bookone">
       <property name="text">
        <string>Cool Blue</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="booktwo">
       <property name="text">
        <string>Berserk!!!!</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="bookthree">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="book1">
       <property name="text">
        <string>book img</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="book2">
       <property name="text">
        <string>book img2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="book3">
       <property name="text">
        <string>book img3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
